server.js

```diff
@@ -1,33 +1,71 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+import netflixData from './data/netflix-titles.json';
+
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
+
+// Two routes here
+
+//First route filter once on multiple fields based on title OR country OR desription OR genre
+app.get('/api', (req, res) => {
+  // query parameter
+  const searchString = req.query.search;
+  let filteredApi = netflixData;
+
+  if (searchString) {
+    filteredApi = filteredApi.filter(item => {
+      const itemTitle = item.title.toString();
+      const itemCountry = item.country.toString();
+      const itemDescription = item.description.toString();
+      const itemGenre = item.listed_in.toString();
+      return (
+        itemTitle.includes(searchString.toLowerCase()) ||
+        itemCountry.includes(searchString) ||
+        itemDescription.includes(searchString) ||
+        itemGenre.includes(searchString)
+      );
```

Comment:
nice to be able to search freely regardless if its a country or title and so on...